app/admin/members.rb

```diff
@@ -72,10 +72,9 @@ def new_unloggable_member.email_required?
       respond_with_dual_blocks(new_unloggable_member, options, &block)
     end
 
-    def update(&block)
+    def update
       resource.skip_reconfirmation!
-      super do |success, failure|
-        block&.call(success, failure)
+      super do |failure|
         failure.html { render :edit }
```

Comment:
You have to use either `|format|` or `|success, failure|` (see [inherited resources](https://github.com/activeadmin/inherited_resources#overwriting-actions))
Here, `failure.html { render :edit }` will actually be rendered whether the update is a failure or not